Gemfile.lock

```diff
@@ -311,69 +312,73 @@ GEM
       actionpack (>= 5.0)
       railties (>= 5.0)
     rexml (3.2.5)
-    rubocop (1.26.1)
+    rubocop (1.35.1)
+      json (~> 2.3)
       parallel (~> 1.10)
-      parser (>= 3.1.0.0)
+      parser (>= 3.1.2.1)
       rainbow (>= 2.2.2, < 4.0)
       regexp_parser (>= 1.8, < 3.0)
-      rexml
-      rubocop-ast (>= 1.16.0, < 2.0)
+      rexml (>= 3.2.5, < 4.0)
+      rubocop-ast (>= 1.20.1, < 2.0)
       ruby-progressbar (~> 1.7)
       unicode-display_width (>= 1.4.0, < 3.0)
-    rubocop-ast (1.16.0)
+    rubocop-ast (1.21.0)
       parser (>= 3.1.1.0)
-    rubocop-performance (1.13.3)
+    rubocop-performance (1.14.3)
       rubocop (>= 1.7.0, < 2.0)
       rubocop-ast (>= 0.4.0)
     ruby-progressbar (1.11.0)
     rubyzip (2.3.2)
-    selenium-webdriver (4.1.0)
+    selenium-webdriver (4.4.0)
       childprocess (>= 0.5, < 5.0)
       rexml (~> 3.2, >= 3.2.5)
-      rubyzip (>= 1.2.2)
-    sidekiq (6.4.1)
-      connection_pool (>= 2.2.2)
+      rubyzip (>= 1.2.2, < 3.0)
+      websocket (~> 1.0)
+    sidekiq (6.5.6)
+      connection_pool (>= 2.2.5)
       rack (~> 2.0)
-      redis (>= 4.2.0)
+      redis (>= 4.5.0, < 5)
     simple_form (5.1.0)
       actionpack (>= 5.2)
       activemodel (>= 5.2)
-    sshkit (1.21.2)
+    sshkit (1.21.3)
       net-scp (>= 1.1.2)
       net-ssh (>= 2.8.0)
     sshkit-interactive (0.3.0)
       sshkit (~> 1.12)
-    standard (1.9.1)
-      rubocop (= 1.26.1)
-      rubocop-performance (= 1.13.3)
-    stimulus-rails (1.0.4)
+    standard (1.16.1)
+      rubocop (= 1.35.1)
+      rubocop-performance (= 1.14.3)
+    stimulus-rails (1.1.0)
       railties (>= 6.0.0)
-    strscan (3.0.1)
+    strscan (3.0.4)
     thor (1.2.1)
-    timeout (0.2.0)
-    turbo-rails (1.0.1)
+    timeout (0.3.0)
+    turbo-rails (1.1.1)
       actionpack (>= 6.0.0)
+      activejob (>= 6.0.0)
       railties (>= 6.0.0)
-    tzinfo (2.0.4)
+    tzinfo (2.0.5)
       concurrent-ruby (~> 1.0)
-    unicode-display_width (2.1.0)
+    unicode-display_width (2.3.0)
     warden (1.2.9)
       rack (>= 2.0.9)
     web-console (4.2.0)
       actionview (>= 6.0.0)
       activemodel (>= 6.0.0)
       bindex (>= 0.4.0)
       railties (>= 6.0.0)
-    webdrivers (5.0.0)
+    webdrivers (5.1.0)
```

Comment:
### [webdrivers](https://github.com/titusfortner/webdrivers)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 5.0.0 → 5.1.0

[_[change-log](https://github.com/titusfortner/webdrivers/blob/master/CHANGELOG.md), [source-code](https://github.com/titusfortner/webdrivers/tree/v5.1.0), [gem-diff](https://my.diffend.io/gems/webdrivers/5.0.0/5.1.0)_]





<details>
<summary>Commits</summary>

A change of **10** commits. See the full changes on [the compare page](https://github.com/titusfortner/webdrivers/compare/v5.0.0...v5.1.0).

These are the individual commits:
- (972103f) [Add Ruby 3.1 to CI (#228)](https://github.com/titusfortner/webdrivers/commit/972103fe1f48f752f6601823f2057caa3860e92a)
- (161d1bb) [Improve error messages for debugging (#232)](https://github.com/titusfortner/webdrivers/commit/161d1bb67fbf74638dcc82068cc6c002a7caed89)
- (14c30e6) [Fix circular require between webdrivers.rb and railtie.rb (#…](https://github.com/titusfortner/webdrivers/commit/14c30e69f99b2c37fc06b5fff3a54c49a02d3d06)
- (cf7959b) [Fix spec for chromedriver beta version](https://github.com/titusfortner/webdrivers/commit/cf7959baec4439d879fcf55d285aef6de450210c)
- (75dae08) [Fix spec for edgedriver beta version](https://github.com/titusfortner/webdrivers/commit/75dae089b1ac731aa5aea45bb4d2c2861b1bd2b2)
- (9c9f11f) [Download correct geckdriver file on Apple Silicon (#234)](https://github.com/titusfortner/webdrivers/commit/9c9f11f8a750b60732a9cdfdc95c37e8f23e8869)
- (a45e4d1) [Add /opt/chromium.org/chromium to Chromium locations for Lin…](https://github.com/titusfortner/webdrivers/commit/a45e4d10929f5ef76d87383e10e8503a8c076afe)
- (f37cc82) [Fix line length](https://github.com/titusfortner/webdrivers/commit/f37cc82ff21cd09515b0b91d141772b4ade22ea2)
- (f05f362) [add support for Microsoft Edge on Linux](https://github.com/titusfortner/webdrivers/commit/f05f362825e7ad35e48c07235a1e286a9ce386f4)
- (268827f) [update changelog and bump version to 5.1.0](https://github.com/titusfortner/webdrivers/commit/268827f2f35f3796f9054fb136e176c407cea53f)


</details>